src/Maestro/maestro-angular/src/index.prod.html

```diff
@@ -0,0 +1,6 @@
+<html lang="en">
+<head>
+</head>
+<body>
+</body>
```

Comment:
? What's this file.  It's just an empty HTML file.

Reply:
This is an empty file to let angular inject its styles and js into. The C# thing can then extract the injected stuff easily.